### PR TITLE
Make edx-lint commands work with both python2 and python3

### PR DESCRIPTION
**Pull request by cpennington**
@nedbat, @clintonb: I had to make these changes to make it so that the IDA cookiecutter could run in both python2 and python3 virtualenvs.

### CODE REVIEW

**Comment by clintonb** [comment]
:+1:

File: edx_lint/tamper_evident.py [hunk 1]
```diff
@@ -26,21 +26,21 @@ def write(self, text, hashline=b"# {}"):
         be changed to accommodate different file syntaxes.
 
         Arguments:
-            text (byte string): the contents of the file to write.
+            text (text string): the contents of the file to write.
```
Comment (nedbat) [inline edx_lint/tamper_evident.py]: Let's get off on the right Python 3 foot, and say "unicode string", since that's what it is.

**Comment by nedbat** [comment]
:+1: once the type names are changed in the docstring.

File: edx_lint/tamper_evident.py (hunk 1 above)
Comment (cpennington) [inline edx_lint/tamper_evident.py]: Ok. I tend to refer to the distinction as text vs bytes, and have seen others do the same, but I'm fine w/ unicode as well.

**Pull request merged**